helios-services/src/main/java/com/spotify/helios/master/MasterParser.java

```diff
@@ -101,4 +112,26 @@ protected void addArgs(final ArgumentParser parser) {
   public MasterConfig getMasterConfig() {
     return masterConfig;
   }
+
+  private static class VersionStringForAuthArgumentChoice implements ArgumentChoice {
+
+    @Override
+    public boolean contains(Object val) {
+      String value = String.valueOf(val);
+      if ("all".equalsIgnoreCase(value)) {
```

Comment:
either is fine; maybe squeezing the two things (enable or a version string) into one arg is too much